hw/ip/aes/doc/_index.md

```diff
@@ -285,6 +338,27 @@ After finishing operation, software must:
 3. Clear the previous input data by overwriting the Input Data registers {{< regref "DATA_IN0" >}} - {{< regref "DATA_IN3" >}}.
 4. Clear the internal key registers and the Output Data register by setting the KEY_CLEAR and DATA_OUT_CLEAR bits in {{< regref "TRIGGER" >}} to `1`.
 
+The code snippet below shows how to perform this task.
+
+```c
+  // Disable autostart
+  REG32(AES_CTRL(0)) = 0x1 << AES_CTRL_MANUAL_START_TRIGGER;
+
+  // Clear Initial Key registers
+  for (int i = 0; i < 8; i++) {
+    REG32(AES_KEY0(0) + i * 4) = 0x0;
```

Comment:
I still think we need to provide this function as a configuration feature: write `1` to `CFG.CLRKEY` and `1` to `CFG.CLRDATA` to randomize the key and data values. Setting them to zero releases sidechannel information of their Huffman value. WDYT?

Reply:
I totally agree with you. As a first step, the HW should just write 0. Once we have a source for random data, those registers (as well as others) should be overwritten using random values. I will create an issue for this and add this feature.

Reply:
sounds good. perhaps put a quick note to that feature coming as a TODO or just documentation note here.

Reply:
would it make sense to just put that feature in now? The prim_lfsr module is already there, and that really can be used for this purpose.  The random data will likely just seed the lfsr modules differently.

I do think longer term there is maybe an even better solution though..

Reply:
Yes, this sounds good. I will implement this soon but I would like to do some work on the data path first. I created the note in the spec and an issue to track such items #1005.